tests/unit/test-api.c

```diff
@@ -760,8 +759,13 @@ main (void)
                                false);
 
     JERRY_ASSERT (!jerry_value_has_error_flag (res));
-    JERRY_ASSERT (jerry_value_is_undefined (res));
+    JERRY_ASSERT (jerry_value_is_string (res));
+    sz = jerry_get_string_size (res);
+    JERRY_ASSERT (sz == 20);
+    sz = jerry_string_to_char_buffer (res, (jerry_char_t *) buffer, sz);
+    JERRY_ASSERT (sz == 20);
     jerry_release_value (res);
+    JERRY_ASSERT (!strncmp (buffer, "string from snapshot", (size_t) sz));
 
```

Comment:
Both global mode and eval mode will return the value of the last js statement. Is it what we expected?